EtlPipelineBlobTrigger/course_stats.py

```diff
@@ -1,37 +1,49 @@
 """Data transformation code for statistical data."""
 
-import json
 import inspect
+import json
 import os
 import sys
 import unicodedata
 from collections import OrderedDict
 
+from validators import validate_agg, validate_unavailable_reason_code
+
 CURRENTDIR = os.path.dirname(
     os.path.abspath(inspect.getfile(inspect.currentframe())))
 sys.path.insert(0, CURRENTDIR)
 
-from validators import validate_unavailable_reason_code, validate_agg
-
 
 class CourseStats:
     def get_stats(self, raw_course_data):
         stats = {}
 
         continuation = Continuation()
         employment = Employment()
-        stats['continuation'] = continuation.get_continuation(raw_course_data)
-        stats['employment'] = employment.get_employment(raw_course_data)
+        salary = Salary()
```

Comment:
We could keep the stat objects in alphabetical order

Reply:
:+1: